lib/srv/db/postgres/sql/redshift-delete-user.sql

```diff
@@ -3,8 +3,8 @@ LANGUAGE plpgsql
 AS $$
 BEGIN
     -- Only drop if the user doesn't have other active sessions.
-    IF EXISTS (SELECT usename FROM pg_stat_activity WHERE usename = username) THEN
-        RAISE NOTICE 'User has active connections';
+    IF EXISTS (SELECT user_name FROM stv_sessions WHERE user_name = CONCAT('IAM:', username)) THEN
```

Comment:
Nit: Maybe include a link to https://docs.aws.amazon.com/redshift/latest/dg/r_STV_SESSIONS.html that explains what this table is? I had to look it up.